lisp/mastodon-tl.el

```diff
@@ -512,13 +513,33 @@ LINK-TYPE is the type of link to produce."
      'keymap mastodon-tl--link-keymap
      'help-echo help-text)))
 
+(defun mastodon-tl--handle-to-account (handle)
```

Comment:
Name this something like "search account by handle"?  And place this in `mastodon-profile.el`?